data/aecore/.minerva/accounts_uat.json

```diff
@@ -0,0 +1,4 @@
+{
+   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000002
```

Comment:
I see you changed the amount, so probably possible

Can you please check if we still own the private key and set a huge amount?

Reply:
Dincho, this is just a dummy address to prove fork works as expected. Provided with the correct address - we can seed it

Reply:
Just generate (a couple of) new keypair(s) and give the private key(s) to Dincho ;-)